index.html

```diff
@@ -47,6 +47,7 @@
             <li id="nav-section"><a href="#about">About</a></li>
             <li id="nav-section"><a href="#faq">FAQ</a></li>
             <li id="nav-section"><a href="#sponsors">Sponsors</a></li>
+            <li id="nav-section"><a href="#meettheteam">Meet The Team</a></li>
```

Comment:
change the href from #meettheteam to /meet-the-team/index.html to link to that page.